website/src/pages/fdc3-roadmap.js

```diff
@@ -0,0 +1,160 @@
+import Layout from "@theme/Layout";
+import Container from "../components/Container"
+const React = require('react');
+
+
+const style = `
+table.roadmap img {
+  max-width: none;
+}
+
+table.roadmap td {
+  text-align: center;
+}
+
+table.roadmap th {
+  text-align: left;
+}
+
+div.centered {
+  margin: auto;
+  padding: 30px;
+}
+`
+
+export default (props) => {
+  return (
+    <Layout title="FDC3 Roadmap">
+      <Container>
+        <style>
+          {style}
+        </style>
+
+        <div class="centered">
+
+          <h1>FDC3 2.1 Roadmap</h1>
+          <p>This roadmap aims to track the standardization streams and
+            activities of the FDC3 2.1 effort.</p>
+
+          <table class="roadmap">
+            <thead>
+              <tr>
+                <th colspan="2"></th>
+                <th>Intents and Context Data</th>
+                <th>Desktop Agent Bridging</th>
+                <th>Identity</th>
+              </tr>
+            </thead>
+            <tr>
+              <th colspan="5"><h3>Initiation</h3></th>
+            </tr>
+            <tr>
+              <th colspan="2">Issue Raised</th>
+              <td>✅</td>
+              <td>✅</td>
+              <td><a href="https://github.com/finos/FDC3/issues/684">Issue
+                Raised</a></td>
+            </tr>
+            <tr>
+              <th colspan="2">Discussed at Standards Working Group</th>
+              <td>✅</td>
+              <td>✅</td>
+              <td></td>
+            </tr>
+            <tr>
+              <th colspan="2">Discussion Group Formed</th>
+              <td>✅</td>
+              <td>✅</td>
+              <td></td>
+            </tr>
+            <tr>
+              <th colspan="2">Maintainers Responsible</th>
+              <td>Vinay Mistry</td>
+              <td>Kris West</td>
+              <td>tbd (Symphony?)</td>
```

Comment:
Should we know if Symphony is going to be the maintainer for the Identity workstream before publishing this page?